frontends/web/src/locales/en/app.json

```diff
@@ -491,9 +501,21 @@
       "title": "What does incoming mean?"
     },
     "accountInfo": {
+      "multipleXPubs": {
+        "text": "Each xpub is tied to the \"Type\" shown: either \"Native segwit (bech32)\" or \"Wrapped segwit\". These are script types used by {{coinName}}. The BitBoxApp combines them, supporting multiple script types in the same account. But because each script type gives a different xpub, there are multiple xpubs per account.\n\nIf you consistently receive on the standard address, you only need the \"bech32\" xpub. However, if you also receive funds on the \"compatible address\" type, these will be shown on the \"older segwit\" extended public key.",
```

Comment:
> But because

afaik starting a sentence in english with 'But' is not proper grammar, you can just remove the word and it works the same way. same below.